workspaces/marketplace/plugins/marketplace-backend/src/router.ts

```diff
@@ -38,7 +39,16 @@ export async function createRouter({
   router.use(express.json());
 
   router.get('/plugins', async (_req, res) => {
-    const plugins = await marketplaceApi.getPlugins();
+    const { cursor, limit, sortByField, sortOrder, searchText } = _req.query;
+    const plugins = await marketplaceApi.getPlugins({
+      cursor: typeof cursor === 'string' ? cursor : undefined,
+      limit: typeof limit === 'string' ? limit : undefined,
+      sortByField: typeof sortByField === 'string' ? sortByField : undefined,
+      sortOrder:
+        sortOrder === 'asc' || sortOrder === 'desc' ? sortOrder : SortOrder.asc,
+      searchText: typeof searchText === 'string' ? searchText : undefined,
+    });
+
```

Comment:
As we discussed, can you please support multiple filters and orders here, like:

* GET ... `?filters[kind]=component&filters[user]=all` for filters
* GET ... `?order=name,ASC&order=title,ASC` for the order
* GET ... `?searchText=...` 

And it would be great to have an offset as well.